x/ccv/provider/keeper/validator_set_update.go

```diff
@@ -229,5 +196,14 @@ func (k Keeper) FilterValidators(
 // GetLastBondedValidators iterates the last validator powers in the staking module
 // and returns the first MaxValidators many validators with the largest powers.
 func (k Keeper) GetLastBondedValidators(ctx sdk.Context) ([]stakingtypes.Validator, error) {
```

Comment:
In the ADR you write:
> For the first iteration, we propose to increase the active set size to 200 validators (while keeping the consensus validators to 180), thus letting the 20 validators with the most stake outside of the active set validate on consumer chains.

I was wondering. How are we going to set the max validators? Is the idea to go through a gov proposal to increase it on the `staking` module or are we going to do it directly as part of the migration?

Reply:
Gov prop imo, but good question. @mpoke wdyt?

Reply:
From the [signaling prop for this feature](https://www.mintscan.io/cosmos/proposals/930):
> We plan to mitigate this risk by going in small steps and, for the first iteration, only allowing the first 20 validators outside the Cosmos Hub’s active validator set to validate on consumer chains. We expect that this will still be a relatively competitive set.

So I think we don't need another gov prop. We will have a software upgrade proposal for Gaia v20 detailing all the changes.

Reply:
> coordination with block explorers (e.g., mintscan, etc.) is needed so they do not start presenting validator sets in an erroneous way.

This is true. I'll add it as a TODO.

Reply:
Good point, thanks for clarifying